lib/src/svg/theme.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:ui';
+
+/// A theme used when decoding an SVG picture.
+class SvgTheme {
+  /// Instantiates an SVG theme with the [colorFilter] and [currentColor].
+  const SvgTheme({
+    this.colorFilter,
+    this.currentColor,
+  });
+
+  /// The color filter, if any, to apply to this widget.
+  final ColorFilter? colorFilter;
```

Comment:
This seems confusing to add here - it's not used quite the same way as `currentColor`, and I'd rather not encourage people to use `colorFilter` since it's kind of expensive.

Reply:
Yeah, that makes sense. Will remove `colorFilter` from `SvgTheme`.